public_html/index.php

```diff
@@ -348,9 +348,9 @@
       <div class="card h-100 shadow-sm">
         <div class="card-body d-flex align-items-center">
           <div>
-            <h5 class="card-title">Cloud-ready</h5>
-            <p class="card-text">Pipelines are tested on AWS after every release.
-              You can even browse results live on the website and use outputs for your own benchmarking.
+            <h5 class="card-title">Robust</h5>
+            <p class="card-text">Pipelines are tested on data from different parts of the tree of life.
```

Comment:
`the tree of life` or just `Tree of Life`?

Reply:
We can mention viewing the results on Tower and S3 later.

Reply:
This tree of life is the taxonomy not our programme.

Reply:
Yes Tower and S3 are a bit glitchy at the moment. I thought we could mention it once we figure out how to display the results for the pipelines.